Update manual for MySQL, Xenial support drop (#1501)
theforeman/theforeman.org: _includes/manuals/nightly/3.5.3_database_setup.md

> Foreman is a rails application. While it is possible to use different databases, Foreman only supports PostgreSQL for production deployments.

Reviewer: ```suggestion
Foreman is a Rails application. While Rails supports various databases, Foreman only supports PostgreSQL for production deployments.
```

The database configuration file can be found at:

<pre>/etc/foreman/database.yml</pre>

> #### PostgreSQL

Reviewer: Is this header still needed if just before we state it's the only supported production deployment?

Author: good point, removed and reworded the next sentence.

Foreman is a Rails application. While Rails supports different databases, Foreman supports only PostgreSQL for production deployments.

The database configuration file can be found at:

<pre>/etc/foreman/database.yml</pre>

<div class="alert alert-info">When using PostgreSQL, you should make sure that the

> foreman-postgresql package installed. See <a href="manuals/{{page.version}}/index.html#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>

Reviewer: ```suggestion
foreman-postgresql package is installed. See <a href="manuals/{{page.version}}/index.html#3.3InstallFromPackages">3.3 Install From Packages</a>.</div>
```

Edit your config/database.yml and modify:

> {% highlight ruby %}

Reviewer: Perhaps unrelated, but technically it's yaml instead of ruby :)
